app/assets/javascripts/models/beta/story.js

```diff
@@ -129,10 +130,9 @@ export const deleteStory = (storyId, projectId) =>
   httpService
     .delete(`/projects/${projectId}/stories/${storyId}`);
 
-export const updateStory = (story, newAttributes) => ({
-  ...story,
-  ...newAttributes
-});
+export const addNewAttributes = (current, newAttributes) => {
+  return extend({}, { ...current, ...newAttributes });
```

Comment:
We don't even need `extend` here anymore :smile: , we can use only spread:

```js
return { ...current, ...newAttributes };
```

Reply:
Yeah, it's sound better. Thank you!